### ISSUE TITLE
Feature requests

### ISSUE THREAD

**Issue by XXLRay** [pixelb/fslint#119]
First of all - I love fslint. It's very convenient to use and discovered many dead backups I would have overlooked. Nevertheless I encountered some things that may make it even more comfortable to use.

- I made a scan over all my hard drives and found a backup folder. Instead of removing the files one by one I used my file browser. Unfortunately the files still display in FSlint. It would be great to have an update option only for the files that have already been detected without the need to run a time extensive full search again.

- I found a different version of a directory. It was easy to decide which version I want to keep but very time intensive to delete every single file. It would be nice to have an option that lets you execute the choice for all matches in the identified folders.

- Some file types that I didn't need to be shown were displayed after a search. Instead searching again (for a loooong time) it would be nice to exclude file types (and/or certain folders/files) from the current display if you have forgotten (or didn't know about them in advance) to exclude them before the search.

tldr: Allow for a display update of the found matches without the need to run a full search again

**Comment by XXLRay** [comment]
I forgot one thing.

- When renaming a file the match is automatically removed from display even if I wanted to remove the other file.

**Comment by XXLRay** [comment]
Yet another one ...

- It would be great to have an option to save and load certain search profiles.

**Comment by XXLRay** [comment]
Me again. I found the option to select all files from a folder but does it check if the reference folder is the same?

**Comment by XXLRay** [comment]
Another thing:

- When I select multiple files so that not all of them fit into the display window at the same time and delete them the display jumps to the very bottom afterwards. That way it's hard to keep track of where I was before the action.

**Comment by pixelb** [comment]
All very good suggestions thanks. The last display focus issue, sounds like a bug that should be easy enough to fix. As for selecting all files in a folder, you can select using path wildcards like /the/full/path/*

**Comment by XXLRay** [comment]
Thanks for the answer and for the hint.

**Comment by tenleftfingers** [comment]
I'd like to add a request for;
1) Scanning mounted samba shares for duplicates, and
2) If for some reason the application can't scan, notify the user rather than tease them for hours with "searching..." :)

**Comment by pixelb** [comment]
errors should be propagated to the gui interface immediately, so I'll need to look into further why samba shares may not be supported.

**Comment by dandv** [comment]
> I found the option to select all files from a folder but does it check if the reference folder is the same?

Would #157 address this?
